hotel_reservation_summary_month/wizard/hotel_reservation_summary.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from datetime import date
+
+from odoo import api, models
+from odoo.fields import Date
+
+
+class RoomReservationSummary(models.TransientModel):
+    _name = "hotel.room.reservation.summary.month"
+    _inherit = "hotel.room.reservation.summary"
+    _form_view = (
+        "hotel_reservation_summary_month"
+        ".hotel_room_reservation_summary_month_form_view"
+    )
+
+    @api.multi
+    @api.depends("date_from")
+    def _compute_name(self):
+        for summary in self:
+            month = Date.from_string(summary.date_from).strftime("%B")
+            summary.name = month
+
+    @api.model
+    def action_generate_summary(self):
+        first = date.today().replace(day=1)
+        next_month = (first.month + 1) % 12
+        next_first = first.replace(month=next_month)
+        wizard = self.create(
+            {
+                "date_from": Date.to_string(first),
+                "date_to": Date.to_string(next_first),
+            }
+        )
+        return wizard.get_summary_action()
+
+    @api.multi
+    def next_month_action(self):
+        self.ensure_one()
+        date_from = Date.from_string(self.date_from)
```

Comment:
Again, not sure why this is done here. Probably because of the JS?

Reply:
Same here, version 11.0 returns strings for dates. :-)